payment/utils.py

```diff
@@ -7,13 +7,15 @@
 from django.core.mail import EmailMultiAlternatives
 from django.utils.translation import ugettext_lazy as _
 from django.utils import translation
+from opaque_keys.edx.keys import CourseKey
 
 from requests.exceptions import ConnectionError
-
 from edxmako.shortcuts import render_to_string
-from student.models import CourseEnrollment
 
 from openedx.core.djangoapps.commerce.utils import ecommerce_api_client
+from openedx.core.djangoapps.course_groups.models import CourseUserGroup
+from openedx.core.djangoapps.course_groups.cohorts import get_cohort_by_name, \
+    add_user_to_cohort
```

Comment:
Not a Black format?
```suggestion
from openedx.core.djangoapps.course_groups.cohorts import (
    get_cohort_by_name, add_user_to_cohort,
)
```

Reply:
Not all my changes are black-formatted, unfortunately, because I did not want to make too many changes to the edited modules. But I can certainly make that particular change.

Reply:
I feel you 😅